doc/man3/SSL_CTX_set_cert_comp_cb.pod

```diff
@@ -0,0 +1,129 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_cert_comp_preference,
+SSL_set_cert_comp_preference,
+OSSL_cert_comp_fn,
+SSL_CTX_set_cert_comp_cb,
+SSL_set_cert_comp_cb,
+SSL_get_cert_to_compress,
+SSL_set1_compressed_cert - Certificate compression functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_CTX_set_cert_comp_preference(SSL_CTX *ctx, uint16_t *algs, int num_algs);
+ int SSL_set_cert_comp_preference(SSL *ssl, uint16_t *algs, int num_algs);
+
+ typedef int (*OSSL_cert_comp_fn)(SSL *ssl, uint16_t alg, void *arg);
+ int SSL_CTX_set_cert_comp_cb(SSL_CTX *ctx, OSSL_cert_comp_fn cert_comp_cb, void *arg);
+ int SSL_set_cert_comp_cb(SSL *ssl, OSSL_cert_comp_fn cert_comp_cb, void *arg);
+
+ int SSL_get_cert_to_compress(SSL *ssl, BUF_MEM *buf_mem);
+ int SSL_set1_compressed_cert(SSL *ssl, uint16_t algorithm, BUF_MEM *buf_mem);
```

Comment:
Why a BUF_MEM rather than just `unsigned char *` and length?

Reply:
Personally, I find it easier to manipulate and pass around. Separate pointers and lengths are so yesterday.